### DISCUSSION TITLE
Support `SignedInteger` where `RawValue: SignedInteger`

### DISCUSSION THREAD

**Discussion by benjohnbarnes** [pointfreeco/swift-tagged#81] [Q&A]
In a current project I am using `Tagged` as the `Interval` of a type `Note` conforming to `Strideable`. This works great as far as it goes. However, I have a `ClosedRange<Note>` on which I would like to be able to compute `count`. This requires `Tagged` conform to `SignedInteger where RawValue: SignedInteger`. 

`Tagged` has a number of conditional conformances and does conditionally support `Numeric` and `SignedNumeric`, there currently is not support for `SignedInteger` as far as I can tell. I'm curious if this was intentionally left out, or if it would be reasonable to add?

**Comment by stephencelis** [comment]
@benjohnbarnes We'd be down for a pull request if you'd like to submit one! Because this is an additive feature and not a bug, though, I'm going to convert to a discussion for now.

**Reply by benjohnbarnes** [reply]
Hello – just saw your reply a month later! I've [added a PR with the changes](https://github.com/pointfreeco/swift-tagged/pull/82). Thanks.

**Reply by benjohnbarnes** [reply]
Hello – Bump. Thanks!
